CSS/contact-us style/form.css

```diff
@@ -0,0 +1,117 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100%;
+    margin-top: 2rem;
+}
+
+.form__container {
+    background: #ffffff;
+    padding: 5rem;
+    border-radius: 2rem;
+    box-shadow: 0 8px 16px rgba(0, 0, 0, 0.1);
+    width: 90%;
+    max-width: 70rem;
+}
+
+input[type="text"],
+input[type="email"],
+input[type="password"],
+input[type="tel"],
+textarea {
+    border: 0.1rem solid #000000;
+    margin-bottom: 1rem;
+    padding: 1rem;
+    border-radius: 1rem; 
+}
+
+#contactLogo {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+#contactLogo img {
+    max-width: 7.5rem;
+}
+
+#contactLogo p {
+    font-family: 'Helvetica', sans-serif;
+    font-size: 3rem;
+    font-weight: bold;
+    color: #333;
+    text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.3)
+}
+
+.form__inputGroup {
+    padding-bottom: 2rem;
+}
+
+.form__inputGroup img {
+    max-height: 1.5rem;
+}
+
+label {
+    font-size: 1.5rem;
+    padding: 0.5rem 0;
+    cursor: text;
+}
+
+input, textarea {
+    width: 100%;

```

Comment:
Around line 30 the input and textarea fields are already targeted and assigned some values. 

Take that into consideration

Reply:
Right that line is actually useless calculating that the befault input style has already that values